cmd/ctr/commands/run/run_unix.go

```diff
@@ -45,11 +47,11 @@ import (
 )
 
 var platformRunFlags = []cli.Flag{
-	&cli.StringFlag{
+	&cli.StringSliceFlag{
 		Name:  "uidmap",
 		Usage: "Run inside a user namespace with the specified UID mapping range; specified with the format `container-uid:host-uid:length`",
```

Comment:
I think the usage string (for this and gidmap) needs to reflect that it is now a slice/list of maps; at least the first clause should end with **mapping range(s)**; could also add at the end a new clause: ", separate multiple mappings with a comma"

Reply:
After this change, the help message for this option becomes the following. 
```
--uidmap container-uid:host-uid:length [ --uidmap container-uid:host-uid:length ]  Run inside a user namespace with the specified UID mapping range; specified with the format container-uid:host-uid:length
```
I think the additional `[]` will indicate the option is a slice. But I think updating the "mapping range" to "mapping ranges" was a good call out. I will make the update in the next revision.